images/ubuntu/scripts/ubicloud/setup-runner-user.sh

```diff
@@ -100,4 +100,9 @@ ACTIONS_RUNNER_HOOK_JOB_COMPLETED=/home/runner/actions-runner/complete-hook.sh"
 echo "PATH=$PATH" >> ./actions-runner/.env
 
 mv ./actions-runner /home/runner/
+
+# Optimizes the runner process startup time
+# https://github.com/actions/runner/blob/6ca97eeb88810dc898e04d53efcd80d8b24bc4e4/src/Runner.Listener/Runner.cs#L193
+/home/runner/actions-runner/bin/Runner.Listener warmup && rm -rf /home/runner/actions-runner/_diag
```

Comment:
Why do we need to rm /home/runner/actions-runner/_diag ?

Reply:
It contains a lot of lines showing the names of loaded classes. The runs without warmup don’t have these. Since we use the same image for all jobs, the logs will always look the same, so those lines aren't really necessary.